hash/pc98_cd.xml

```diff
@@ -1195,6 +1264,43 @@ license:CC0
+		<rom name="Chuushaki - Hirasawa Nurse School (Japan) (Track 03).bin" size="21732480" crc="32dc10a7" sha1="f5a2a5e063cf5549355850a3112bda625238d77f"/>
+		<rom name="Chuushaki - Hirasawa Nurse School (Japan) (Track 04).bin" size="22527456" crc="26778a44" sha1="562390d337a8be82cd7fb7a6d5d92c97b7e61217"/>
+		<rom name="Chuushaki - Hirasawa Nurse School (Japan) (Track 05).bin" size="22706208" crc="e4319540" sha1="617c79f9751ff6f6c39d0eb5930f54d6f96b4693"/>
+		<rom name="Chuushaki - Hirasawa Nurse School (Japan) (Track 06).bin" size="21450240" crc="6ba8cdfc" sha1="f9d0f63971d449e9183ff6024d128026977c2343"/>
+		<rom name="Chuushaki - Hirasawa Nurse School (Japan) (Track 07).bin" size="23214240" crc="f8d832e5" sha1="a32b11f613b3aa6dbec72122102031c27560fa70"/>
+		<rom name="Chuushaki - Hirasawa Nurse School (Japan) (Track 08).bin" size="22221696" crc="9aed99e8" sha1="2718ed4f918ba0ccd94e582de525f9ae38c07a9f"/>
+		<rom name="Chuushaki - Hirasawa Nurse School (Japan) (Track 09).bin" size="22266384" crc="ba398e9b" sha1="beea65e885176018b3f69f2a1014b1f4e219f0cb"/>
+		<rom name="Chuushaki - Hirasawa Nurse School (Japan) (Track 10).bin" size="22346352" crc="8f96a5c8" sha1="7742d3c8d5a307b3fcd33f47d38cec35555abdf6"/>
+		<rom name="Chuushaki - Hirasawa Nurse School (Japan) (Track 11).bin" size="21833616" crc="b441e1b8" sha1="8f4456bc51134b3d20c061a28cb95d8035473d15"/>
+		<rom name="Chuushaki - Hirasawa Nurse School (Japan) (Track 12).bin" size="24082128" crc="0df65c34" sha1="d368de83a7437d466e8a8f3f3168461e96ba84f0"/>
+		<rom name="Chuushaki - Hirasawa Nurse School (Japan) (Track 13).bin" size="21518448" crc="c58a089e" sha1="b5e310da088d34d64c28be88e58daa3d75e0e94a"/>
+		<rom name="Chuushaki - Hirasawa Nurse School (Japan) (Track 14).bin" size="22050000" crc="7623bbe4" sha1="df17ac4b18ccb5f58aed71817ff36bdb5a59c6d1"/>
+		<rom name="Chuushaki - Hirasawa Nurse School (Japan) (Track 15).bin" size="23334192" crc="7ef323dc" sha1="335d283a97c548bfb8054efa5576f654b859c04b"/>
+		<rom name="Chuushaki - Hirasawa Nurse School (Japan) (Track 16).bin" size="23146032" crc="0478e8e7" sha1="8387b6455db2147a78c208e8e9f115adbc9f277a"/>
+		<rom name="Chuushaki - Hirasawa Nurse School (Japan) (Track 17).bin" size="23397696" crc="1181caba" sha1="4a1ed0c2e7e49c53bdf7138bca2571bb6e7cd5ee"/>
+		<rom name="Chuushaki - Hirasawa Nurse School (Japan) (Track 18).bin" size="23364768" crc="2faeed4e" sha1="f90cab13a35a3e780e4c35e8c01ec5086bf31bd1"/>
+		<rom name="Chuushaki - Hirasawa Nurse School (Japan) (Track 19).bin" size="5386080" crc="50f2c593" sha1="da0a7a8c5dabcfaa1870878459b5877c92e78eab"/>
+		<rom name="Chuushaki - Hirasawa Nurse School (Japan).cue" size="2175" crc="5f3f2a6f" sha1="568d26425c5437d0389d2378eb25631303ff8ded"/>
+		 -->
+		<description>Chuushaki - Hirasawa Nurse School</description>
+		<year>1996</year>
+		<publisher>アーヴォリオ (Aypio)</publisher>
+		<info name="alt_title" value="注射器" />
```

Comment:
Is this the full Japanese title?

Reply:
See cover art here: https://vndb.org/v8633

The subtitle seems to be part of the full title even though it’s written in Latin script.